docs/ops/opset10.md

```diff
@@ -0,0 +1,185 @@
+# opset9 {#openvino_docs_ops_opset9}
+
+This specification document describes the `opset9` operation set supported in OpenVINO™.
+Support for each particular operation from the list below depends on the capabilities of an inference plugin
+and may vary among different hardware platforms and devices. Examples of operation instances are provided as IR V10 xml
+snippets. Such IR is generated by the Model Optimizer. The semantics match corresponding nGraph operation classes
+declared in `namespace opset9`.
+
+
+## Table of Contents <a name="toc"></a>
```

Comment:
Please add: 
- opset10 to: [docs/ops/opset.md](https://github.com/openvinotoolkit/openvino/pull/12162/commits/605d6d25a0a36b09b5b7a72714c9db8ff6b3f9f5)

- Unique op to: [/docs/OV_Runtime_UG/Operations_specifications.md](https://github.com/openvinotoolkit/openvino/blob/94ef65f0bd07221f79f5a57c7dbdc389eb2e09f9/docs/OV_Runtime_UG/Operations_specifications.md)